### PR TITLE
Convert bootstrap.csx to bat file...

### PR DESCRIPTION
**Pull request by KevinH-MS**
If the csi on my machine gets corrupted by patching, I'd still like the
bootstrap to work so I can get a csi and install.csx back to a good state.

This means we'll need an install.sh as well, but the script is simple enough
that I think that's fine.

### CODE REVIEW

**Comment by KevinH-MS** [comment]
FYI @TyOverby 
/cc @rchande @basoundr @VSadov

**Comment by TyOverby** [comment]
LGTM

File: src/Test/Perf/bootstrap.bat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@echo off
+
+call ..\..\..\Restore.cmd
```
Comment (TyOverby) [inline src/Test/Perf/bootstrap.bat]: Don't use relative paths.  Bootstrap should be callable from anywhere, not just from the same directory that bootstrap.bat is in.

File: src/Test/Perf/bootstrap.bat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+set MSBuild=%ProgramFiles%\MSBuild\14.0\bin\msbuild.exe
+if not exist "%MSBuild%" set MSBuild=%ProgramFiles(x86)%\MSBuild\14.0\bin\msbuild.exe
+"%MSBuild%" ..\..\Interactive\csi\csi.csproj /p:Configuration=Release /p:OutDir="%~dp0infra\bin\\"
```
Comment (TyOverby) [inline src/Test/Perf/bootstrap.bat]: Same issue here.

**Comment by TyOverby** [comment]
You'll want to add infra/bin to the .gitignore file.  (We have a seperate .gitignore in the Perf directory)

File: src/Test/Perf/bootstrap.bat (hunk 1 above)
Comment (KevinH-MS) [inline src/Test/Perf/bootstrap.bat]: Good point!  Fixed

File: src/Test/Perf/bootstrap.bat (hunk 2 above)
Comment (KevinH-MS) [inline src/Test/Perf/bootstrap.bat]: Fixed

**Comment by TyOverby** [comment]
Looks good!

**Pull request merged**